### ISSUE TITLE
Feat: move all documentation on external website (community website)

### ISSUE THREAD

**Issue by PhilippeAccorsi** [tracim/tracim#2912]
## Feature description

All documentation can be easly accessible and we can see admin, user and dev documentation in same place.

## Context

At present some documentation is accesible here doc.tracim.fr (user documentation), and in repo but in different folder (docker, frontend, backend)

## Impacted components

*"unknown"*

## Extra information

**Comment by grignards** [comment]
@gbonaspetti @PhilippeAccorsi Following what we discussed yesterday I propose to remove this issue from 3.1.0 (put it in 3.3.x?) and instead put a new one explaining to do this issue tasks in order (?) for 2 days maximum + check the done tasks in this issue.

**Comment by PhilippeAccorsi** [comment]
Like a meta issue ? I think its a good idea.

**Comment by grignards** [comment]
Yes, the idea is to keep this issue as a meta one (since we decided we won't do all tasks for 3.1.0).

**Comment by PhilippeAccorsi** [comment]
@PhilippeAccorsi

**Comment by PhilippeAccorsi** [comment]
Branch `misc/2912__improve_documentation_architecture` used to improve documentation.
